### PR TITLE
[8] Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by KutrCB**
## Task: `Adding Page Navigation`
- `page navigation
screen111
`

### CODE REVIEW

File: submissions/charlesBanda/html-forms/Contact.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/charlesBanda/html-forms/Contact.html:1]: I noticed in your [7] task, you've added this file, you don't have to added it here, you can remove this file from this branch.

Comment (KutrCB) [reply submissions/charlesBanda/html-forms/Contact.html:1]: Yes, i worked on the signup page and login page. should i add them?

Comment (ToluAkin) [reply submissions/charlesBanda/html-forms/Contact.html:1]: NO. Don't add the signup page and login page. They already exist. Your [7] pull request has been merged. You need to delete this file to resolve the conflict on this branch.
@KutrCB

**Comment by KutrCB** [comment]
That is why i had pushed  updated (submissions/charlesBanda/html-forms/Contact.html) and index.html that contain the the linkage. The idea was to push the two pages because the contain the link to each other

**Pull request closed without merging**